app/serializers/encryptable_minimal_serializer.rb

```diff
@@ -1,9 +1,18 @@
 # frozen_string_literal: true
 
 class EncryptableMinimalSerializer < ApplicationSerializer
-  attributes :id, :name, :description, :created_at, :sender_name
+  attributes :id, :name, :description, :created_at, :sender_name, :used_attrs
```

Comment:
```suggestion
  attributes :id, :name, :description, :created_at, :sender_name, :used_encrypted_data_attrs
```
I know, longer term like this, but more clear and specific